src/revivalpmmp/pureentities/entity/monster/walking/Wolf.php

```diff
@@ -309,9 +309,9 @@ public function tame(Player $player): bool {
 
             // set the properties accordingly
             $this->setTamed(true);
-            $this->setCollarColor(self::RED); // default color
             $this->setOwner($player);
             $this->setSitting(true);
```

Comment:
When wolf is tamed, not get Sitting automaticaly. Need add you to sitting. @kniffo80 . Test on singleplayer.